code/datums/abilities/werewolf.dm

```diff
@@ -97,7 +97,7 @@
 
 			M.visible_message("<span class='alert'><B>[M] [pick("metamorphizes", "transforms", "changes")] into a werewolf! Holy shit!</B></span>")
 			if (M.find_ailment_by_type(/datum/ailment/disease/lycanthropy))
-				boutput(M, "<span class='notice'><h3>You are now a werewolf.</span></h3>")
+				boutput(M, "<span class='alert'><h2>You are not an antagonist!</h2> Your transformation was achieved by in-game means, you have the form of a werewolf but are <i>not</i> an antagonist.</span></h3>")
```

Comment:
They might still be an antagonist that used the serum.

Reply:
Yeah I kinda just copied the vampire one, didn't really think about this.

Reply:
X